docs/source/coremltools.optimize.torch.examples.rst

```diff
@@ -0,0 +1,9 @@
+Training-Time Compression Examples
+=====================
+
+.. toctree::
+    :maxdepth: 1
+
+    ../_examples/magnitude_pruning.rst
```

Comment:
Is the sphinx going to interpret those `.py` files as `.rst` while compiling the documentation?
I think we can do:
1. Move the examples under `coremltools/optimize/torch/` folder
2. In this file, we use the absolute path `coremltools.optimize.torch.examples.magnitude_pruning.rst` .. etc
This might be more clean?

Reply:
1. So I think we should separate examples from the source code. Maybe we can do `examples/optimize/torch` or `examples/coremltools/optimize/torch`
2. These `rst` files are auto-generated. The location of the source python files and the destination rst files are in `docs/conf.py`

Reply:
Moved to `docs/examples/optimize/torch`. Marking as resolved.